kong/templates/nginx.lua

```diff
@@ -23,17 +23,44 @@ events {
 > end
 }
 
-> if wasm and wasm_modules_parsed and #wasm_modules_parsed > 0 then
+> if wasm then
+load_module $(wasm_dynamic_module);
+
 wasm {
+  env RUST_BACKTRACE=1;
+  env WASMTIME_BACKTRACE_DETAILS=1;
```

Comment:
These can't go inside `wasm {}`, but honestly they should just get removed, because they are just dev artifacts:

```suggestion
```

Reply:
related https://github.com/Kong/kong/pull/11233